examples/compareKernelTypes.py

```diff
@@ -49,70 +51,70 @@
 defSciencePath = None
 defTemplatePath = None
 
+
 class DiffimTestCases(unittest.TestCase):
```

Comment:
Oy. As with many of the examples throughout the stack, these may not run at all. Try to run it and see what happens, otherwise maybe delete it? Most of the code is from ~2009.

(looks at code in the actual file)

Woah! It's even got test suite stuff in it. Making it a test that runs would be nice (more tests is better), but that might be a "fun" exercise.